hack/install_kustomize.sh

```diff
@@ -1,10 +1,24 @@
 #!/bin/bash
 
-# Downloads the most recently released kustomize binary
-# to your current working directory.
+# If no argument is given -> Downloads the most recently released
+# kustomize binary to your current working directory.
+# (e.g. 'install_kustomize.sh')
+#
+# If an argument is given -> Downloads the specified version of the
+# kustomize binary to your current working directory.
+# (e.g. 'install_kustomize.sh 3.8.2')
 #
 # Fails if the file already exists.
 
+if [ -z "$1" ]
+  then
+    echo "No version specified. Downloading the most recently released kustomize binary."
```

Comment:
No echo statements except on error.
Other scripts use this, and expect no output on success (as with almost all unix commands).

Reply:
I removed the unnecessary echo statements.